config.js

```diff
@@ -51,5 +51,10 @@ module.exports = {
       'shooting-clubs': 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=shootclub/',
       museums: 'https://eforms.homeoffice.gov.uk/outreach/Feedback.ofml?FormName=Museums/'
     }
+  },
+  pdf: {
+    url: env !== 'production' ?
+    `http://${process.env.PDF_CONVERTER_HOST}:${process.env.PDF_CONVERTER_PORT}/convert` :
+    process.env.PDF_CONVERTER_URL
```

Comment:
It possibly looks like this is for docker-compose, but could we not just configure docker compose to have `PDF_CONVERTER_URL=http://html-pdf-converter:8001/convert` (since that's what it resolves to anyway) and dispense with the need for conditionals here?